partner_pricelist_search/models/res_partner.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class Pricelist(models.Model):
+    _inherit = "res.partner"
+
+    property_product_pricelist = fields.Many2one(
+        search="_search_property_product_pricelist"
+    )
+
+    @api.model
+    def _search_property_product_pricelist(self, operator, value):
+        if operator == "=":
```

Comment:
I think you can use the same technique as used in other module. @CarlosRoca13 can you point to the module with the operator import?

Reply:
Yes, sure, here you have the PR were was added that logic https://github.com/OCA/sale-reporting/pull/117

Reply:
After seeing it, it's not as easy and this code is enough.